pom.xml

```diff
@@ -38,14 +38,14 @@
                                 <include>**/src/**/*</include>
                                 <include>**/res/**/*</include>
                                 <include>**/lib/**/*</include>
-                                <include>**/bundle.json</include>
+                                <include>**/bundle.js</include>
```

Comment:
As a second thought, I'm going to say this is a non-issue; platform has completely migrated.  Other projects using this build file should update accordingly.